deployment/build_and_release/modules/cloudbuild_triggers/main.tf

```diff
@@ -0,0 +1,45 @@
+locals {
+  cloudbuild_path = ({
+    "prod" = "release/cloudbuild.yaml"
+    "ci" = "release/cloudbuild_ci.yaml"
+    "presubmit" = "release/cloudbuild_presubmit.yaml"
+  })
+}
+
+resource "google_service_account" "builder" {
+  account_id   = "cloudbuild-${var.env}"
+  display_name = "Armored Witness ${var.env} Builder Service Account"
+}
+
+resource "google_cloudbuild_trigger" "applet_release" {
+  location = "global"
+  # service_account = google_service_account.builder.id
+
+  github {
+    owner = "transparency-dev"
+    name  = "armored-witness-applet"
+
+    dynamic "push" {
+      for_each = var.env == "prod" ? [1] : []
```

Comment:
Each of these triggers being different depending on the environment suggests that this is 3 different modules to me. At a bare minimum, this shouldn't be switching on the environment name, but an explicit parameter that enumerates the type of trigger to set up (e.g. `ON_TAG`, `PUSH_TO_MAIN`, `PULL_REQUEST`).